### ISSUE TITLE
Adding new languages - specifcally hebrew

### ISSUE THREAD

**Issue by shgidi** [JaidedAI/EasyOCR#363]
Hi @rkcosmos , thank you for your great work!

I really liked the modularity, however, I did not find in the issues anyone that was able to reproduce your pipeline. I wonder if it is intentional. 
I went through the process you've suggested and was able to load Hebrew model I trained with (most of) the process you described.
Will you be interested in adding this model to your model zoo (along with a pull request for code changes)? If not it is also fine :)

**Comment by rkcosmos** [comment]
yes, I can do that. Please give me a link to your model and a PR.

**Comment by rkcosmos** [comment]
I cannot read Hebrew but I heard there are complications like niqqud and t’amim. Does your model support niqqud and/or t’amim?

**Comment by shgidi** [comment]
Thank you for your reply. I'll make some more experiments to improve the model.
Can you share some data generation parameters you've used? I currently try a diverse number of words and sizes. I use 100K texts for training.
Re the **niqqud** and **t'amim** issues - My model doesn't include them, but I don't think we should be concerned about them - They don't exist in 99% of Hebrew text, and even if they are, people mostly ignore them. They only exist in texts such as
 1. Biblical or ancient texts 
 2. Educational texts for small children  
 3. Poetry

**Comment by imvladikon** [comment]
@shgidi yeap, I wrote it a lot of times ago [here](https://github.com/JaidedAI/EasyOCR/issues/91#issuecomment-658917506)  also sent vocab for training. I participate to request, it will be useful to get train part. or description of it in terms using data generation part (https://github.com/Belval/TextRecognitionDataGenerator). it will be helpful

**Comment by imvladikon** [comment]
also geresh/gershayim are important, wrote it [here](https://github.com/JaidedAI/EasyOCR/issues/91) but it's possible to replace them with just a quotes mark as it happened in usual life.

**Comment by berrytsakala** [comment]
1. Where could we follow the status for hebrew?

2. @shgidi can you / would publish your data publically?

3. Shgidi is of course correct about niqqud & taaming. Historically speaking, they're just a recent "plugin" to the language, and besides - they are the 1% that will consume 99% of the effort.

**Comment by shgidi** [comment]
@berrytsakala Unfortunately this project got a little bit off our focus. We are hoping to get back to it later this year.
Generally, it's not very hard to do - just need to follow instructions in the repo's for generating data and training, find out the right config, and solve some bugs that come along the way. We reached a reasonable model with some hacking but didn't find the time to complete the task. I'll update this repo if and we'll progress with this.

**Comment by azoulaynoam** [comment]
Is the Hebrew support available?

**Comment by thesmartwon** [comment]
Just tried `he`, `heb`, and `iw` and they do not work. Please reopen this issue or explain why it was closed.

**Comment by danhertztech** [comment]
Any update on whether EasyOCR will be adding support for Hebrew, and if so, when? Thank you.

[https://www.jaided.ai/easyocr/](https://www.jaided.ai/easyocr/)

**Comment by 3eliran** [comment]
Is it available?
